tests/functional/iceberg/test_table_basic.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+
+from pathlib import Path
+
+from dbt.tests.util import run_dbt, rm_file
+
+_MODEL_BASIC_TABLE_MODEL = """
+{{
+  config(
+    materialized = "table",
+    cluster_by=['id'],
+  )
+}}
+select 1 as id
+""".strip()
```

Comment:
The only reason I've seen to use `.strip()` in these fixtures is for inline CSVs; the extra return gets read as an empty record. It doesn't hurt for models, but it's also not necessary.

Reply:
Oh sure, force of habit at work. Removed.